### ISSUE TITLE
Make the seq_length large?

### ISSUE THREAD

**Issue by guoxiaolu** [harvitronix/five-video-classification-methods#67]
Brilliant job, I have noticed that when extract_feature, the seq_length is given as 40. Besides, the min frame number is about 50 in UCF dataset. So, it is reasonable to set that value lower than 50. 

In line 228, of 'data.py', 'assert len(input_list) >= size', that means seq_length larger than 50 is not permitted although it is useful for final acc.

Do you know the normal method of how to set a large seq_length larger than the frame number? Set the all zero features to the end of the sequence?

**Comment by harvitronix** [comment]
@guoxiaolu I think what you'll want to do is essentially what you said. Initialize the seq_length to zeros, and then set your frames at the start or end, or simply pad your sequence with however many spaces you need to create arrays of equal length.

Take a look at Kera's `pad_sequences` method: https://keras.io/preprocessing/sequence/

**Comment by guoxiaolu** [comment]
Thank you very much, it is what I want to know.

**Comment by Ido-PX** [comment]
@guoxiaolu did you end up padding empty frames? did it improve your accuracy?
